workflow/airflow/dags/local_file_sensor_dag.py

```diff
@@ -0,0 +1,94 @@
+import os
+
+import airflow
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.sensors import BaseSensorOperator
+from datetime import timedelta
+
+
+class FileWatcherOperator(BaseSensorOperator):
```

Comment:
suppose I want to write a test for this class. Which options do I have? Unit tests by stubbing lots of methods or integration tests using a real airflow process?

Reply:
In the next PR I will add some components for building DAGs (small module of reusable operators) along with their own tests. 

Unit tests won't need that much stubbing, after stubbing a DAG, the config and context passed around they are just Python functions and classes.

I would also have some integration tests for confidence that an entire workflow does what we think. There is a fairly lightly documented `TEST_MODE` you can put an `airflow` instance into which might help, would need to investigate this feature further.